src/applications/vre/28-1900/containers/IntroductionPage.jsx

```diff
@@ -111,6 +111,27 @@ const IntroductionPage = props => {
       <div className="omb-info--container" style={{ paddingLeft: '0px' }}>
         <OMBInfo resBurden={15} ombNumber="2900-0009" expDate="11/30/2022" />
       </div>
+      <h4>To apply by mail</h4>
```

Comment:
I think the headers in this file may not be in sequential order; I only see `h4` through `h6` in this file.

Reply:
@Mottie The H1, H2 and H3 on this page are provided by the form title and the save in progress content